### PR TITLE
New DC to check for Gene Symbol Classifier in Ensembl cores

### PR DESCRIPTION
**Pull request by leannehaggerty**
This will check that the GSC has been run on cores that have ensembl annotations (it will skip anything that is not 'anno' or 'full_genebuild')

I am still in the process of handing over the cores that previously failed in the gene symbol classifier pipeline, once handed over there shouldn't be anything on sta-5 that fails the DC

### CODE REVIEW

File: lib/Bio/EnsEMBL/DataCheck/Checks/GeneSymbols.pm [hunk 1]
```diff
@@ -0,0 +1,56 @@
+=head1 LICENSE
+Copyright [2018-2022] EMBL-European Bioinformatics Institute
```
Comment (marcoooo) [inline lib/Bio/EnsEMBL/DataCheck/Checks/GeneSymbols.pm:2]: License should be 2023.

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: lib/Bio/EnsEMBL/DataCheck/Checks/GeneSymbols.pm [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+     if($method ne 'anno' && $method ne 'full_genebuild') {
+         skip "Gene Symbols not mandatory for non-Genebuild annotations", 1;
+     }
+
+     my $desc = 'Gene Symbol Transformer has been run';
+     my $sql  = qq/
+      SELECT COUNT(*) FROM
+        analysis
+      WHERE
+        logic_name LIKE 'gene_symbol_classifier'
```
Comment (ens-LCampbell) [inline lib/Bio/EnsEMBL/DataCheck/Checks/GeneSymbols.pm:50]: Is there any other variation of logic name related to gene_symbolc_classifier expected ?  'gene_symbol_classifier%' ? 

If not, would be better to be explicit:
logic_name = 'gene_symbol_classifier'.

File: lib/Bio/EnsEMBL/DataCheck/Checks/GeneSymbols.pm (hunk 1 above)
Comment (leannehaggerty) [reply lib/Bio/EnsEMBL/DataCheck/Checks/GeneSymbols.pm]: Yep, sorry fixed that

**Review by ens-LCampbell — APPROVED** [review]

File: lib/Bio/EnsEMBL/DataCheck/Checks/GeneSymbols.pm (hunk 2 above)
Comment (leannehaggerty) [reply lib/Bio/EnsEMBL/DataCheck/Checks/GeneSymbols.pm:50]: We're likely going to add more specific classifiers, i.e. the current is trained on verts but we're working on a non-verts model now. I'll need to update this DC in the future either way

**Review by ens-LCampbell — APPROVED** [review]
Code checks out for me, happy to approve merge

**Review by marcoooo — APPROVED** [review]

**Pull request merged**